packages/react/src/index.ts

```diff
@@ -9,7 +9,7 @@ export type {
   WithClerkProp,
   WithSessionProp,
 } from './types';
-export { isMagicLinkError, MagicLinkErrorCode } from './errors';
+export { MagicLinkErrorCode, isClerkAPIResponseError, isKnownError, isMetamaskError, isMagicLinkError } from './errors';
```

Comment:
❓ Why not export them directly from the `@clerk/shared` instead of `./errors` ?

Reply:
It just seemed cleaner to have all error related stuff in one place. Does the other way around seem better to you?